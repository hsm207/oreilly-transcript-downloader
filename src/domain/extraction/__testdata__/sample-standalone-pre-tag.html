<!--
  Source: Standalone <pre> tag test for BookChapterExtractor
-->
<div id="sbo-rt-content">
  <div id="_idContainer011">
    <h1 id="_idParaDest-35" class="chapter-number"><a id="_idTextAnchor035"></a>2</h1>
    <h1 id="_idParaDest-36">
      <a id="_idTextAnchor036"></a>Generating Text with AI for Content Creation
    </h1>
    <p><span class="No-Break">Prompt:</span></p>
    <div data-testid="custom pre block" class="orm-ChapterReader-codeSnippetContainer">
      <pre class="source-code">Write a Tweet about being a solopreneur</pre>
      <div class="orm-ChapterReader-snippetButtonContainer"></div>
    </div>
    <p><span class="No-Break">Response:</span></p>
    <div data-testid="custom pre block" class="orm-ChapterReader-codeSnippetContainer">
      <pre class="console">
🚀 Embracing the #solopreneur life! 🌟 Juggling all roles, making decisions, and learning every day. It's a wild ride, but I'm loving the freedom and growth it brings. 👩‍💻📈 Shoutout to my fellow solopreneurs, let's show 'em what we're made of! 💪 #EntrepreneurSpirit #BossingItUp 🌍</pre
      >
      <div class="orm-ChapterReader-snippetButtonContainer"></div>
    </div>
    <p>Now, let’s make it <span class="No-Break">a thread.</span></p>
  </div>
</div>
