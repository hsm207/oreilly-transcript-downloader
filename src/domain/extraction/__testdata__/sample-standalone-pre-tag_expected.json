[
  {
    "type": "heading",
    "level": 1,
    "text": "2"
  },
  {
    "type": "heading",
    "level": 1,
    "text": "Generating Text with AI for Content Creation"
  },
  {
    "type": "paragraph",
    "text": "Prompt:"
  },
  {
    "type": "preformatted",
    "text": "Write a Tweet about being a solopreneur"
  },
  {
    "type": "paragraph",
    "text": "Response:"
  },
  {
    "type": "preformatted",
    "text": "🚀 Embracing the #solopreneur life! 🌟 Juggling all roles, making decisions, and learning every day. It's a wild ride, but I'm loving the freedom and growth it brings. 👩‍💻📈 Shoutout to my fellow solopreneurs, let's show 'em what we're made of! 💪 #EntrepreneurSpirit #BossingItUp 🌍"
  },
  {
    "type": "paragraph",
    "text": "Now, let's make it a thread."
  }
]
